main/User_config.h

```diff
@@ -80,6 +80,10 @@ const byte ip[] = {192, 168, 1, 99};
 const byte mac[] = {0xDE, 0xED, 0xBA, 0xFE, 0x54, 0x95}; //W5100 ethernet shield mac adress
 #endif
 
+#ifndef NTP_SERVER
+#  define NTP_SERVER "pool.ntp.org"
+#endif
```

Comment:
Should we have this for Arduino UNO and Mega also

Reply:
I didn't think it necessary since it only defines a constant. It can be moved inside a conditional check if you prefer.

Reply:
ok good point